install/j2eemgmt/bin/ts.jte

```diff
@@ -66,7 +66,7 @@ env.ts_unix.command.testExecuteAppClient=com.sun.ts.lib.harness.ExecTSTestCmd DI
         LD_LIBRARY_PATH=${J2EE_HOME}/nativelib \
 	TZ=${TZ} \
         ${JAVA_HOME}/bin/java -Dorg.omg.CORBA.ORBInitialHost=${ORB_HOST} \
-		${JAVA_OPTIONS} -Djava.security.policy=${TS_HOME}/bin/java.policy -Djava.util.logging.config.file=${J2EE_HOME}/config/client_logging.properties \
+		${JAVA_OPTIONS} -Djava.security.policy=${TS_HOME}/bin/java.policy -Djava.util.logging.config.file=${TS_HOME}/bin/client-logging.properties \
```

Comment:
It should be ok to remove the changes in this file as it is not used for Platform TCK run.

Reply:
It was rather defensive change, I plan to take a look later again what is used and what isn't, first it must be really simple to verify changes. It's on my stack, it just must not overflow :-)